lnbits/wallets/lnpay.py

```diff
@@ -139,25 +136,26 @@ async def paid_invoices_stream(self) -> AsyncGenerator[str, None]:
             yield value
 
     async def webhook_listener(self):
+        logger.error("LNPay webhook listener disabled.")
+        return
         # TODO: request.get_data is undefined, was it something with Flask or quart?
         # probably issue introduced when refactoring?
-        text: str = await request.get_data()  # type: ignore
-        try:
-            data = json.loads(text)
-        except json.decoder.JSONDecodeError:
-            logger.error(f"got something wrong on lnpay webhook endpoint: {text[:200]}")
-            data = None
-        if (
-            type(data) is not dict
-            or "event" not in data
-            or data["event"].get("name") != "wallet_receive"
-        ):
-            raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
-
-        lntx_id = data["data"]["wtx"]["lnTx"]["id"]
-        r = await self.client.get(f"/lntx/{lntx_id}?fields=settled")
-        data = r.json()
-        if data["settled"]:
-            await self.queue.put(lntx_id)
-
-        raise HTTPException(status_code=HTTPStatus.NO_CONTENT)
+        # text: str = await request.get_data()
```

Comment:
dead code?

Reply:
lets open a issue for those 2, this code does not work currently thats why i commented it out, at some point some need to reimplement it i guess.